src/Tests/AssertEx.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Compilation.Parser.Binding.Parser;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace DotVVM.Framework.Tests
+{
+    public static class AssertEx
+    {
+        public static void AssertNode(BindingParserNode node, string expectedDisplayString, int start, int length, bool hasErrors = false)
```

Comment:
I'd rename this to `BindingNode`, it's on longer in "binding" class, but it's now in "assert" class.

Reply:
Done